.github/workflows/run_staging_deployment.yml

```
@@ -0,0 +1,39 @@
+---
+name: Deploy To Staging
+
+on:
+  workflow_run:
+    workflows: ['Create pre-release and push Docker image']
+    branches: [main]
+    types:
+      - completed
+  workflow_dispatch:
+
+
+permissions:
+  contents: read
+
+jobs:
+  create_new_release:
```

```suggestion
  deploy_to_staging:
```
Dont know if this something we could change but maybe changing the name of the job